airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -491,9 +491,9 @@ def on_kill(self) -> None:
 
     def build_pod_request_obj(self, context=None):
         """
-        Creates a V1Pod based on user parameters. Note that a `pod` or `pod_template_file`
-        will supersede all other values.
+        Returns V1Pod object based on pod template file, full pod spec, and other operator parameters.
 
+        Full pod spec overrides pod template, and operator params take highest precedence.
```

Comment:
Would reverse the order so that the most important variable is mentioned first:

```suggestion
        The V1Pod attributes are selected from (first to last): operator params, full pod spec, pod template file.
```